### PR TITLE
add --no-header flag and hide unused persistent flags

### PR DESCRIPTION
**Pull request by wlan0**

### CODE REVIEW

File: functests/common.sh [hunk 1]
```diff
@@ -194,7 +194,8 @@ function uninstall_minio() {
     "${DIRECT_CSI_CLIENT}" volumes ls
 
     while true; do
-        count=$("${DIRECT_CSI_CLIENT}" volumes ls | grep -v WARNING| wc -l)
+        count=$("${DIRECT_CSI_CLIENT}" volumes ls | grep -vc WARNING)
```
Comment (balamurugana) [inline functests/common.sh:197]: You should use ` | wc -l` otherwise `grep -vc` will return zero on failure.

File: functests/tests.sh [hunk 2]
```diff
@@ -94,7 +94,7 @@ function do_upgrade_test() {
     # Show output for manual debugging.
     "${DIRECT_CSI_CLIENT}" volumes list --all -o wide
 
-    mapfile -t upgraded_volumes < <("${DIRECT_CSI_CLIENT}" volumes list --status published | awk '{print $1}')
+    mapfile -t upgraded_volumes < <("${DIRECT_CSI_CLIENT}" volumes list --status published | grep -v WARNING | awk '{print $1}')
```
Comment (balamurugana) [inline functests/tests.sh:97]: Use `awk '/WARNING/ {print $1}'` instead of `grep`

File: functests/tests.sh [hunk 3]
```diff
@@ -117,7 +117,7 @@ function do_upgrade_test() {
...
```
Comment (balamurugana) [inline functests/tests.sh:120]: Same as above

File: functests/common.sh (hunk 1 above)
Comment (wlan0) [reply functests/common.sh:197]: `grep | wc -l` leads to lint failure.

Comment (balamurugana) [reply functests/common.sh:197]: > `grep | wc -l` leads to lint failure. 

We need add to ignore list

**Comment by Praveenrajmani** [comment]
Reopened as https://github.com/minio/directpv/pull/491 with the CI fixes and reviews addressed

**Pull request closed without merging**